resources/views/service/traders/edit.blade.php

```diff
@@ -6,8 +6,13 @@
         <div id="main-content">
 
             <h1>Edit Trader</h1>
+            <p>Edit "{{ $trader->name }}" and it's users or market</p>
```

Comment:
❓ **Why the quotation marks?**
```suggestion
            <p>Edit "{{ $trader->name }}" and its users or market</p>
```

Reply:
I wanted the market name to be quoted because there's not really any way we can predict what whacky names they come up with. perhaps *bold* or _italic_?

Reply:
I'd opt of _italics_ then.